packages/gasket-plugin-cypress/README.md

```diff
@@ -4,10 +4,22 @@ Creates a `cypress` based testing environment for your Gasket application.
 
 ## Installation
 
-This plugin should only be used during the create command for new apps.
```

Comment:
Ooh, this should only be used in a preset now for scaffolding with `create-gasket-app`. Perhaps the installation section should be one line:

```
This plugin is only used by presets for `create-gasket-app` and is not installed for apps.
```

This would also be the case for jest, lint, mocha, and typescript plugins.